app/src/services/subscribers/CensusEventSubscriberService.ts

```diff
@@ -80,13 +80,14 @@ export default class CensusEventSubscriberService implements ServiceInterface {
 
     // eslint-disable-next-line @typescript-eslint/require-await
     public async boot(): Promise<void> {
-        CensusEventSubscriberService.logger.info('Booting EventListenerService... (NOT IMPLEMENTED)');
+        CensusEventSubscriberService.logger.info('Booting EventListenerService...');
+
+        this.constructListeners();
```

Comment:
This isn't void any more?

Reply:
There was a void and an await, as the function returns void(no promise) I thought it was redundant.

Reply:
Alrighty